src/gitlab/test_merge_request.py

```
@@ -0,0 +1,397 @@
+        assert result["id"] == 123
+        assert result["title"] == "Test MR"
+        assert result["state"] == "opened"
+
+    @patch("gitlab.merge_request.requests.get")
+    def test_get_merge_request_detail_api_error(self, mock_get):
+        """测试API请求错误"""
+        mock_response = Mock()
+        mock_response.json.side_effect = Exception("JSON decode error")
+        mock_get.return_value = mock_response
+
+        with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)